main/webapp/modules/core/scripts/views/data-table/column-header-ui.js

```diff
@@ -69,7 +69,35 @@ DataTableColumnHeaderUI.prototype._render = function() {
   elmts.dropdownMenu.on('click',function() {
     self._createMenuForColumnHeader(this);
   });
-  
+
+  var serviceUrl = null;
+  var service = null;
+  var serviceLogo=null;
+  if (this._column.reconConfig) {
+   serviceUrl =this._column.reconConfig.service;
+  }
+  try {
+    
+    if (new URL(serviceUrl)) {
+      service = ReconciliationManager.getServiceFromUrl(serviceUrl);
```

Comment:
Can't we use the same utility function? You introduced it inside the Wikibase extension, but if you introduced it in the main application you would be able to use it in all places, no?

Reply:
Absolutely yes! even I was thinking about the same while I was updating the code.